docs/source/integrations/mobile/ios-fairplay-sdk.md

```diff
@@ -87,125 +86,115 @@ More information about ATS can be found in Apple’s TechNote:
 
 ## Preparation
 
-Before you can use the demo project or framework in your own project, for each instance you will require a URL to correctly prepared content, a content ID for the content (also referred to as assetID in previous versions), and a VUDRM token which must be correctly generated for the type of instance you wish to create. Example VUDRM token type templates available are:
+The demo project is based on Apples FairPlay SDK example application. This is configured using the projects `Streams.plist` file. 
 
-- Fairplay Rental `{"type": "r","duration_rental": 3600}`
- - Fairplay Rental tokens may only be used to stream online content to devices running iOS 9.x or higher.
-- Fairplay Persist `{"type": "p","duration_persist": 3600}`
- - Fairplay Persist tokens may be used to stream online content and play offline (downloaded) content to devices running iOS 10.x or higher.
+To provide complete configuration for each `Stream` object in the `Streams.plist`you will require:
+
+ - An `skd://` URI entry in the `content_key_id_list`. The correct URI can be obtained by retrieving the manifest and parsing out the `URI` from the `EXT-X-SESSION-KEY` or `EXT-X-KEY`, this can be done in code or manually with curl in the Terminal app, eg enter: `curl https://eample.cloudfront.net/example-demo/examplecontent/examplecontent.ism/.m3u8`. 
+ - The `playlist_url` to correctly prepared content.
+ -  A `name` for the content (the correct content ID will be parsed out by VUDRMFairPlay SDK and will always be the last path component of the above `skd://` URI entry).
+ - A boolean value which indicates that the `Stream` object is protected with DRM. This setting would not be needed in scenarios where all content is protected, as all assets can be set as protected by default in the source code. 
+ - A `vudrm-token` which must be correctly generated for the type of instance you wish to create. Example VUDRM token type templates available are:
+	- Fairplay Rental `{"type": "r","duration_rental": 3600}`
+		- Fairplay Rental tokens may only be used to stream online content to devices running iOS 9.x or higher.
+	- Fairplay Persist `{"type": "p","duration_persist": 3600}`
+ 		- Fairplay Persist tokens may be used to stream online content and play offline (downloaded) content to devices running iOS 10.x or higher.
 
 For further information about VUDRM please contact us, or refer to our documentation:
 [https://docs.vualto.com/projects/vudrm/en/latest/VUDRM-token.html](https://docs.vualto.com/projects/vudrm/en/latest/VUDRM-token.html)
 
-## Example framework usage
-
-* Import the VUDRMFairPlay module into your view controller:
+For added convenience, VUDRMFairPlay also bubbles up notifications of errors and progress during the licensing request.
 
-	`import vudrmFairPlay`
-
-* Initialise an instance of VUDRMFairPlay:
+Errors can be intercepted as follows:
 
+    @objc func handleVudrmDidError(_ notification: Notification)
```

Comment:
Code should be in code blocks to make reading easier.

Reply:
In code blocks already, as discussed.